### ISSUE TITLE
initial fork/clone experience w/eclipse: top level artifacts, .gitignore

### ISSUE THREAD

**Issue by dlaboss** [quarks-edge/quarks#7]
I forked quarks and used Eclipse/git to create my workspace and download a clone of my fork and import all of the projects. The top level repo artifacts like build.xml, etc don't show up in the project explorer.
Following an "ant all" there are now many untracked files reported because "/target" isn't in the .gitignore.  Ditto for /test/svt/lib.

**Comment by dlaboss** [comment]
pull#9 addressed the gitignore issue.

**Comment by dlaboss** [comment]
As a, hopefully temporary, workaround to make it possible to view/edit the top-level repo artifacts from Eclipse, I did the following in my workspace.
I happen to have my local git repo (~/git/quarks) separate from my workspace so this "links" hackery works:

```
new Project ... > General > "topLevelRepoArtifacts"  (don't share it)
topLevelRepoArtifacts > import ...   > browse to local quarks repo
select each of the top-level files, then >> Advanced  > select "Create links in workspace"
```

One downside to this is that egit often seems to not notice when a top-level artifact has been modified so it doesn't show up in the Git Staging view.  Hence you have to manually `git add ...` these artifacts.

**Comment by vdogaru** [comment]
Add an Eclipse .project file to ~/git/quarks with the following contents seems to work in my environment:

```
<?xml version="1.0" encoding="UTF-8"?>
<projectDescription>
    <name>_quarks</name>
    <comment>Top level project</comment>
</projectDescription>
```

In an empty Eclipse workspace you can import "_quarks" and all the nested Java projects if you do:

```
menu File > Import... > (General) Existing Projects into Workspace > 
Select "Search for nested projects" > Set Select root directory to ~/git/quarks
```

**Comment by ddebrunner** [comment]
I wonder if we can add a top-level project to the repo?

I'm no expert on Eclipse projects, and I'd thought that it didn't like nested projects, but obviously not the case.

So would @vdogaru 's top-level project work with the import directly from git process that is documented in DEVELOPMENT.md

**Comment by vdogaru** [comment]
It does. To successfully import the top project, the "Search for nested projects" option in the Eclipse import dialog box should be selected.  This option is selected by default in the Eclipse Mars 1 release so the user does not have to do extra steps to import the top project as well. In other Eclipse versions this option may not be on by default, so the user needs to select it before importing the projects. I updated DEVELOPMENT.md 'Using Eclipse' instructions, see [pull request 65](https://github.com/quarks-edge/quarks/pull/65).

**Comment by dlaboss** [comment]
fixed by merge #65
